src/databricks/labs/ucx/cli.py

```diff
@@ -165,5 +165,48 @@ def revert_migrated_tables(w: WorkspaceClient, schema: str, table: str, *, delet
         tm.revert_migrated_tables(schema, table, delete_managed=delete_managed)
 
 
+@ucx.command
+def migrate_uc_to_uc(
+    w: WorkspaceClient,
+    from_catalog: str,
+    from_schema: str,
+    from_table: str,
+    to_catalog: str,
+    to_schema: str,
+):
+    """move a uc table/tables from one schema to another schema in same or different catalog"""
+    logger.info("Running move command")
+    installation_manager = InstallationManager(w)
+    installation = installation_manager.for_user(w.current_user.me())
+    if not installation:
+        logger.error(CANT_FIND_UCX_MSG)
+        return
+    sql_backend = StatementExecutionBackend(w, installation.config.warehouse_id)
+    tmp = TableMapping(w, sql_backend)
+    tables = TablesMigrate(
+        TablesCrawler(sql_backend, installation.config.inventory_database),
```

Comment:
Can you extract the variable for table crawler, so that the surrounding constructor call doesn't look odd?